src/helix-ui/elements/_hx-error.less

```diff
@@ -0,0 +1 @@
+@import 'core/hx-icon';
```

Comment:
Double check `margin-right` on the icon.

Reply:
taken out due to adding too much space